libsel4/sel4_plat_include/star64/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2023, Ivan Velickovic
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <autoconf.h>
```

Comment:
Could align this with https://github.com/seL4/seL4/pull/825 an use
```suggestion
#include <sel4/config.h>
```